### PR TITLE
dev to master for 2.1.1

### PR DESCRIPTION
**Pull request by cijothomas**

### CODE REVIEW

File: src/Microsoft.ApplicationInsights.AspNetCore/ApplicationInsightsJavaScript.cs [hunk 1]
```diff
@@ -59,7 +64,7 @@ public string FullScript
                         identity != null &&
                         identity.IsAuthenticated)
                     {
-                        string escapedUserName = identity.Name.Replace("\\", "\\\\");
+                        string escapedUserName = encoder.Encode(identity.Name);
```
Comment (dnduffy) [inline src/Microsoft.ApplicationInsights.AspNetCore/ApplicationInsightsJavaScript.cs]: Please add this change to the changelog notes.

Comment (cijothomas) [inline src/Microsoft.ApplicationInsights.AspNetCore/ApplicationInsightsJavaScript.cs]: Whats the issue # in gh for this change?

Comment (dnduffy) [inline src/Microsoft.ApplicationInsights.AspNetCore/ApplicationInsightsJavaScript.cs]: I'm not sure if we duplicated it on GitHub, it was a security bug from the SDL review.

Comment (dnduffy) [inline src/Microsoft.ApplicationInsights.AspNetCore/ApplicationInsightsJavaScript.cs]: It was from this PR though: https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/411

**Pull request merged**